### PR TITLE
Fix Events API

### PR DESCRIPTION
**Pull request by mihai-dinculescu**
- [x] Fix the Events API
- [x] Add AppMention Example

### CODE REVIEW

File: src/socket/socket_mode.rs [hunk 1]
```diff
@@ -298,13 +300,16 @@ mod test {
   "envelope_id": "dbdd0ef3-1543-4f94-bfb4-133d0e6c1545",
   "accepts_response_payload": false,
   "payload": {
```
Comment (Gompei) [inline src/socket/socket_mode.rs:303]: Could you please add a field? :smiley:
With the current test data, I can't convert it to struct.:cry:

```rust
  "payload": {
    "token": "bHKJ2n9AW6Ju3MjciOHfbA1b",
    "team_id": "T1234567890",
    "api_app_id": "A0000000000",
    "event_id": "Ev0000000000",
    "event_time": 1600000000,
```

```rust
pub struct EventCallback {
    pub token: String,
    pub team_id: String,
    pub api_app_id: String,
    pub event: EventCallbackType,
    pub event_id: String,
    pub event_time: i32,
}
```

**Comment by Gompei** [comment]
@mihai-dinculescu
Thank you for PR:smile:

File: src/socket/socket_mode.rs (hunk 1 above)
Comment (mihai-dinculescu) [reply src/socket/socket_mode.rs:303]: Added.

**Review by Gompei — APPROVED** [review]
LGTM!!

**Pull request merged**